packages/jest-preset-mc-app/load-config.js

```diff
@@ -0,0 +1,48 @@
+const { cosmiconfigSync } = require('cosmiconfig');
+
+const moduleName = 'jest-mc-app';
+const explorer = cosmiconfigSync(moduleName);
+
+const defaultConfig = {
+  silenceConsoleWarnings: [
+    /.*Warning: componentWillReceiveProps has been renamed.*/,
+    /.*CellMeasurerCache should only measure a cell's width or height.*/,
+  ],
+  notThrowWarnings: [
+    /.*@commercetools-frontend\/permissions.*/,
+    /.*Warning: React.createFactory() is deprecated.*/,
+  ],
+};
+
+const mergeSilenceConsoleWarnings = createListMergerWithDefaults(
+  'silenceConsoleWarnings'
+);
+const mergeNotThrowWarnings = createListMergerWithDefaults('notThrowWarnings');
+
+let cachedConfig;
+const loadConfig = () => {
+  if (cachedConfig) {
+    return cachedConfig;
+  }
+  const configFile = explorer.search();
+  if (configFile && configFile.config) {
+    const customConfig = configFile.config;
+    cachedConfig = {
+      ...defaultConfig,
```

Comment:
Should we merge these deep? Is that somewhat what I'd expect as a consumer defining my config overwrites?

Reply:
That was my [question](https://github.com/commercetools/merchant-center-application-kit/pull/1423#discussion_r401711259) as well 🙂